app/services/redmine_request_csv_it.rb

```diff
@@ -0,0 +1,10 @@
+class RedmineRequestCsvIt < RedmineRequestCsv
+  def addition_params
+    [['f[]', 'project_id'],
+     ['op[project_id]', '='],
+     ['v[project_id][]', 129],
+     ['v[project_id][]', 130],
+     ['v[project_id][]', 139],
+     ['v[project_id][]', 140]]
+  end
+end
```

Comment:
I think this should be solved differently than with inheritance because the `addition_params` function in the base class is only present because of the child class.